CHANGELOG.md

```diff
@@ -2,10 +2,9 @@
 
 ### Added
 * Added APIs for Audio Video configuration i.e `AudioVideoConfiguration` to be used during a meeting session.
-* Added support for joining meetings without audio i.e `AudioMode.NoAudio` or with audio using one of `AudioMode.Mono16K`, `AudioMode.Mono48K` and `AudioMode.Stereo48K` audio modes.
+* Added support for joining meetings with audio using one of `AudioMode.Mono16K`, `AudioMode.Mono48K` and `AudioMode.Stereo48K` audio modes.
```

Comment:
Maybe change it to "Added support for joining meetings using one of ...."?

Reply:
Fixed